qemu/tests/cfg/nvdimm.cfg

```diff
@@ -66,3 +66,20 @@
             check_command = "grep -A21 '${nv_backend}' /proc/%s/smaps"
             clean_command = "rm -rf ${mount_dir}/${nv_backend} && umount ${mount_dir} && rm -rf ${mount_dir}"
             start_vm = no
+        - nvdimm_mode:
+            type = nvdimm_mode
+            mem_devs = mem1 mem2
+            mem-path_mem1 = /tmp/test1.img
+            mem-path_mem2 = /tmp/test2.img
+            align_mem1 = 128M
+            align_mem2 = 128M
```

Comment:
Ah~  you add it now.

And, how about `label-size` ?

Reply:
It's in the next line.

Reply:
Yeap,  i miss it. thanks!